ws-client/src/transport.rs

```diff
@@ -227,7 +229,12 @@ impl<'a> WsTransportClientBuilder<'a> {
 					match self.mode {
 						Mode::Plain => TlsOrPlain::Plain(socket),
 						Mode::Tls => {
-							let connector = async_tls::TlsConnector::default();
+							let mut client_config = rustls::ClientConfig::default();
+							if self.use_system_certificates {
+								client_config.root_store = rustls_native_certs::load_native_certs().map_err(|(_, e)| e)?;
```

Comment:
`load_native_certs` is blocking, do we want to `spawn_blocking` or something similar?